tools/install.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+read -p 'Enable ssl? Press Enter for none: ' ssl
+read -p 'Enter your server domain: ' domain
+read -p 'Enter /path/to/exoframe-folder should be path on your server: ' config
+read -p 'Enter /home/user/.ssh/authorized_keys should point to your authorized_keys file: ' ssh
+read -sp 'Enter your private key used for JWT encryption: ' passvar
+
+VAR="docker run -d \
+-v /var/run/docker.sock:/var/run/docker.sock \
+-v $config:/root/.exoframe \
+-v $ssh/authorized_keys:/root/.ssh/authorized_keys:ro \
+-e EXO_PRIVATE_KEY=$passvar \
+--label traefik.enable=true \
+--label traefik.http.routers.exoframe-server.rule=Host(\`exoframe.$domain\`)"
+
+if [ $ssl ]; then
```

Comment:
This should also generate new exoframe-server config in `$config` folder and add `letsencrypt=true` (and letsencrypt email) to it